src/main/java/hudson/plugins/jigomerge/JigomergeBuilder.java

```diff
@@ -27,7 +27,8 @@
 	@Extension
 	public static final JigomergeBuildDescriptor DESCRIPTOR = new JigomergeBuildDescriptor();
 
-	private static final String JIGOMERGE_VERSION = "2.2.6";
+	private static final String JIGOMERGE_VERSION = "2.2.7";
```

Comment:
No need to change the version of jigomerge if you have not changed anything from 2.2.6.